src/common/directives/creditCardCvv.directive.scss

```
@@ -46,3 +46,9 @@
     }
   }
 }
+
+.branded-checkout {
```

```suggestion
branded-checkout
```
I think this might go better in the `branded-checkout.scss` file.